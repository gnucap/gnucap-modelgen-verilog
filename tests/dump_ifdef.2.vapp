
`ifdef AAA
parse error
`else

`ifndef AAA
`define X x
`endif /* another comment */

`endif // another comment

`define M m

model `M(`X,y)
endmodule
