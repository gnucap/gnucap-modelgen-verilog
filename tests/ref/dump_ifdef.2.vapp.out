




model  m ( x ,y)
endmodule
